Server/Ressources/Database/Columns.csv

```diff
@@ -0,0 +1,17 @@
+name;type;table_id
+name;string;0
+price;float;0
+description;string;0
+available_quantity;int;0
+delivery_date;date;0
+image_url;string;0
+id;int;0
+brand;string;0
+category_id;int;0
```

Comment:
If we add the HT price & a reference, some changes will be required here

Reply:
already done